src/VisualStudio/Core/Def/Microsoft.VisualStudio.LanguageServices.csproj

```diff
@@ -16,22 +16,6 @@
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Debug|AnyCPU'" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Release|AnyCPU'" />
-  <!-- We need to mark our NuGet.VisualStudio package reference as EmbedInteropTypes.  However,
-       as this package comes from project.json there is no way to do that in the json file.
-       To deal with that, we write this target which finds that added package, removes it from
-       the references list, then adds it back in again with the 'EmbedInteropTypes' flag properly
-       set. -->
-  <Target Name="MarkNuGetVisualStudioPackageWithEmbedInteropType" AfterTargets="ResolvePackageDependenciesForBuild">
```

Comment:
This commit moves us to the standard NuGet package for doing exclusions, which already does this for NuGet.VisualStudio.